db/schema.rb

```diff
@@ -189,6 +204,12 @@
     t.index ["name"], name: "index_tags_on_name"
   end
 
+  create_table "temp_session_offsets", id: :bigint, default: nil, force: :cascade do |t|
+    t.decimal "latitude"
+    t.decimal "longitude"
+    t.text "timezone"
+  end
+
```

Comment:
Can you check the db migrate status on you machine? It seems there are some old and no needed things.